app/user_auth.py

```diff
@@ -14,10 +14,37 @@
 from lib.database.reservations import get_reservation_for_user
 from lib.database.promotion_keys import pop_promotion_key
 from lib.database.showtimes import get_showtime
+from .lib.email_sender import send_reminder
```

Comment:
this won't import.... look 4 lines below for an example on how to import from `email_sender`